index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <link rel="stylesheet" href="styles.css">
+        <title>first mobile app</title>
+    </head>
+
+    <body>
+        <section class="main-page">
+            <header class="main-header">
+                <nav>
+                    <div class="header-text">John Doe</div>
```

Comment:
- [ ] This div should be an a tag since it is a logo and we should be able to use it to navigate to the index page.

Reply:
done